libr/main/rax2.c

```diff
@@ -311,24 +315,26 @@ static int rax(RNum *num, char *str, int len, int last, ut64 *_flags, int *fm) {
 		ut64 n = r_num_math (num, str);
 		if (n >> 32) {
 			/* is 64 bit value */
-			ut8 *np = (ut8 *) &n;
 			if (flags & 1) {
 				fwrite (&n, sizeof (n), 1, stdout);
 			} else {
-				printf ("%02x%02x%02x%02x"
-					"%02x%02x%02x%02x\n",
-					np[0], np[1], np[2], np[3],
-					np[4], np[5], np[6], np[7]);
+				for (int i = 0; i < 8; ++i) {
```

Comment:
Dont use for(int. Use int;for. Dont define variables inside the for statment

Reply:
Also never use preincrement. Unless its the only way to do it. In this place is unnecessary